content/python.rst

```diff
@@ -140,18 +140,23 @@ NO
   Unicode is a character set and UTF-8 is a particular way of
   encoding Unicode.
 
-* When a method does not use the *self* attribute, this is a code smell
-  that this method should be placed somewhere else.
+* When a method does not use the *self* attribute, this is a sure sign
+  that this method should be placed somewhere else. Also, it's a code smell.
 
 
-Abstract
-========
+Abstract classes - Twisted style
+================================
+
+The descriptions used below for abstract classes follows the 'definition' 
+used by the Twisted framework. While not exactly in line with cannonical Python
+definitions (that mention metaclasses and the use of decorators) these points
```

Comment:
I am not aware of this being a Twisted specific thing...and I think that Twisted would like to see less raise NotImplementedError() code.

I think that is better to use "textboox" instead of Zope or Twisted

Yes, you can observer this pattern in Twisted, but as far as I can tell is only for the code started more than 10 years ago.
In the last 4 years I have not seen this pattern... and people are encouraged to use decorator or composition

> if there's any method decorated as an abstractmethod,

yes. this part is not about abc.abstractmethod

---------

SO for our abstract classes, I would like to keep this section as part of our documentation, mainly so that when someone will maintain such code will know what is it all about.

In the same time, I am encouraging  to always consider composition/decorators/metaclass before writing a **new** abstract class.

So maybe the abstract section can start with the above sentence